src/wara_ps_client.h

```diff
@@ -50,16 +57,12 @@ class WaraPSClient {
     WaraPSClient(std::string name, std::string server_address);
 
     /**
-     * Deleted constructors to disallow copying and moving of the client
+     * Deleted constructors to disallow copying and assignment of the client
      */
-    WaraPSClient(WaraPSClient &&) = delete;
-
     WaraPSClient(const WaraPSClient &) = delete;
 
     WaraPSClient &operator=(const WaraPSClient &) = delete;
```

Comment:
Marking assignment operator as `= delete` will also automatically "remove" the move assignment operator.